MdePkg/Library/StackCheckFailureHookLibNull/StackCheckFailureHook.c

```diff
@@ -0,0 +1,26 @@
+/** @file
+  Library provides a hook called when a stack cookie check fails.

```

Comment:
A hook implies there is a default handler and that a hook can be added to that default behavior.  What is the default behavior when a stack check failure is detected.

Reply:
Hopefully the README lays this out more clearly, if not I can update it. The hooks provides the opportunity for a platform to do any operations required when a stack check failure occurs, for example server platforms may need to send a SEL log. The default behavior is not to do any extra action. Beyond that, it depends on which version of the StackCheckLib you are using on what the behavior is.